knowlg-app/src/app/home/services/content.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import * as _ from 'lodash-es';
+import {data} from './data';
+@Injectable({
+  providedIn: 'root'
+})
+export class ContentService {
+  playerConfig = {};
+  constructor() { }
+  getContenstList(mimeType) {
+    if (mimeType === 'pdf') {
```

Comment:
this should be implemented like `data[mimeType]` not with if conditions

Reply:
Done